content/Joypad_Input.md

```diff
@@ -30,18 +30,18 @@ to the SNES. See SGB description for details.
 
 ### INT 60 - Joypad Interrupt
 
-The Joypad interrupt is requested when any of the above Input lines changes
-from High to Low. This happens when a key is
-pressed (provided that the direction/button key is enabled by
-bit 4/5, respectively), however, because of switch bounce, one or more High to Low
-transitions are usually produced when pressing and releasing a key.
+The Joypad interrupt is requested when any of the bits 0-3 change
+from High to Low. This happens when a button is
+pressed (provided that the action/direction buttons are enabled by
+bit 5/4, respectively), however, due to switch bounce, one or more High to Low
+transitions are usually produced when pressing a button.
 
 ### Using the Joypad Interrupt
 
 It's practically useless for programmers. Even when selecting both
-buttons and direction keys simultaneously it still cannot recognize all
-keystrokes, because in that case a bit might be already held low by a
-button key, and pressing the corresponding direction key would thus
+action and direction buttons simultaneously, it still cannot recognize all
+presses, because in that case a bit might be already held Low by an
```

Comment:
Keystrokes is clearer than presses in my opinion

Reply:
And "button presses"?

Reply:
"each button press" is probably better

Reply:
I have rewritten it to avoid the issue